test/e2e/registration_service_test.go

```diff
@@ -495,6 +495,35 @@ func (s *registrationServiceTestSuite) TestPhoneVerification() {
 
 	// Confirm that VerificationRequired is no longer true
 	require.False(s.T(), mpStatus["verificationRequired"].(bool))
+
+	// Create another token and identity to sign up with
+	otherIdentity := authsupport.NewIdentity()
+	otherEmailValue := uuid.NewV4().String() + "@other.domain"
+	otherEmailClaim := authsupport.WithEmailClaim(otherEmailValue)
+	otherToken, err := authsupport.GenerateSignedE2ETestToken(*otherIdentity, otherEmailClaim)
+	require.NoError(s.T(), err)
+
+	// Call the signup endpoint
+	invokeEndpoint(s.T(), "POST", s.route+"/api/v1/signup", otherToken, "", http.StatusAccepted)
+
+	// Wait for the UserSignup to be created
+	otherUserSignup, err := s.hostAwait.WaitForUserSignup(otherIdentity.ID.String(),
+		wait.UntilUserSignupHasConditions(VerificationRequired()...),
+		wait.UntilUserSignupHasStateLabel(v1alpha1.UserSignupStateLabelValueNotReady))
+	require.NoError(s.T(), err)
+	otherEmailAnnotation := userSignup.Annotations[v1alpha1.UserSignupUserEmailAnnotationKey]
+	assert.Equal(s.T(), otherEmailValue, otherEmailAnnotation)
+
+	// Initiate the verification process using the same phone number as previously
+	invokeEndpoint(s.T(), "PUT", s.route+"/api/v1/signup/verification", otherToken,
```

Comment:
Let's also check the error message returned by this endpoint.

Reply:
Done